src/nodes.coffee

```diff
@@ -1634,6 +1697,12 @@ exports.Code = class Code extends Base
     o.awaitInFile = o.awaitInFile or o_new.awaitInFile
     o.deferInFile = o.deferInFile or o_new.deferInFile
     @icedFlag = o_new.awaitInFunc
+    @icedFoundArguments = o.foundArguments or o_new.foundArguments
+
+    for param in @params
+      if param.name instanceof Literal and param.name.value is iced.const.autocb
+        @foundAutocb = true
```

Comment:
Hm, it's used to throw error when `autocb` is found in parameters, maybe it can `@error` right there instead of setting `foundAutocb`.